src/GitVersion.Core/Git/SemanticVersionBuildMetaData.cs

```diff
@@ -87,15 +87,15 @@ public string ToString(string? format, IFormatProvider? formatProvider)
         if (formatProvider?.GetFormat(GetType()) is ICustomFormatter formatter)
             return formatter.Format(format, this, formatProvider);
 
-        if (format.IsNullOrEmpty())
+        if (string.IsNullOrEmpty(format))
             format = "b";
 
         format = format.ToLower();
         return format.ToLower() switch
         {
             "b" => $"{this.CommitsSinceTag}",
-            "s" => $"{this.CommitsSinceTag}{(this.Sha.IsNullOrEmpty() ? null : ".Sha." + this.Sha)}".TrimStart('.'),
-            "f" => $"{this.CommitsSinceTag}{(this.Branch.IsNullOrEmpty() ? null : ".Branch." + FormatMetaDataPart(this.Branch))}{(this.Sha.IsNullOrEmpty() ? null : ".Sha." + this.Sha)}{(this.OtherMetaData.IsNullOrEmpty() ? null : "." + FormatMetaDataPart(this.OtherMetaData))}".TrimStart('.'),
+            "s" => $"{this.CommitsSinceTag}{(string.IsNullOrEmpty(this.Sha) ? null : ".Sha." + this.Sha)}".TrimStart('.'),
```

Comment:
Why all these changes to `string.IsNullOrEmpty(x)` instead of `x.IsNullOrEmpty()`?

Reply:
I have changed it back